main.py

```diff
@@ -881,7 +881,7 @@ async def close(ctx):
             send_member = await commands.MemberConverter().convert(ctx, user_id)
             dm_channel = await send_member.create_dm()
             embed = discord.Embed(title="Ticket Closed",
-                                  description="The support agent has closed the ticket. Sending a new message will open up a new ticket, so only do so if you have more questions.",
+                                  description="A staff members has closed your ticket. Sending a new message will create a new ticket, please only do so if you have more questions.",

```

Comment:
Shouldn't this be: "A staff member has closed your ticket."?

Reply:
Yeah, apparently my grammar is as bad as yours lol